src/server/index.js

```diff
@@ -29,7 +29,7 @@ server
   .get('/status', (req, res) => {
     res.sendStatus(200);
   })
-  .get('/*', async (req, res) => {
+  .get('/article/*', async (req, res) => {
```

Comment:
We shouldn't be setting routing in Express, but in the [config](https://github.com/bbc/simorgh/blob/latest/src/app/routes/index.js#L3-L9) of [React Router](https://reacttraining.com/react-router/) .

The `/*` route is a catch-all and requests will fall through to React Router, which deals with routing.